cli/internal/cmd/init.go

```diff
@@ -168,12 +172,7 @@ func (i *initCmd) initialize(
 		return fmt.Errorf("reading cluster ID file: %w", err)
 	}
 
-	// config validation does not check k8s patch version since upgrade may accept an outdated patch version.
-	// init only supported up-to-date versions.
-	k8sVersion, err := versions.NewValidK8sVersion(compatibility.EnsurePrefixV(conf.KubernetesVersion), true)
-	if err != nil {
-		return err
-	}
+	k8sVersion := conf.KubernetesVersion
 	i.log.Debugf("Validated k8s version as %s", k8sVersion)
 	if versions.IsPreviewK8sVersion(k8sVersion) {
 		cmd.PrintErrf("Warning: Constellation with Kubernetes %v is still in preview. Use only for evaluation purposes.\n", k8sVersion)
```

Comment:
Having a look at this again, I think we should go with option 2 https://github.com/edgelesssys/constellation/pull/2287#discussion_r1318484238.
I don't think it is readable to have 2 places where we refer to `conf.KubernetesVersion` and validate it in the first part while printing that we've validated it in the second part.

Reply:
@3u13r fixed it